Gemfile

```diff
@@ -7,11 +7,13 @@ group :development do
 end
 
 group :test do
-  gem 'rspec', '~> 3.1'
   gem 'rack-test'
+  gem 'rspec', '~> 3.1'
 end
 
 group :development, :test do
-  gem 'rubocop', '0.28.0'
+  gem 'activerecord', '>= 4.0.0'
+  gem 'mongoid', '>= 5.0.0'
```

Comment:
I think this needs to be optional, eg. multiple gems with appraisals. Some builds will need to run without these an some with either mongoid or pg.